plugins/org.python.pydev/tests_completions/org/python/pydev/ast/codecompletion/revisited/FindActualDefinitionTest.java

```diff
@@ -154,22 +154,22 @@ public void testFindActualDefinitionImported() throws Exception {
     public void testOccurrencesForDefinition() throws Exception {
         int usedGrammar = GRAMMAR_TO_USE_FOR_PARSING;
         GRAMMAR_TO_USE_FOR_PARSING = PythonNature.LATEST_GRAMMAR_PY3_VERSION;
-
-        File file = new File(TestDependent.TEST_PYSRC_TESTING_LOC + "occurrencesForDefinition/foo.py");
-        PySelection selection = new PySelection(new Document(FileUtils.getFileContents(file)), 0, 19, 4);
-        Refactorer.setPyRefactoring(new Refactorer());
-        RefactoringRequest req = new RefactoringRequest(file, selection, nature);
-        req.fillInitialNameAndOffset();
-        req.setAdditionalInfo(RefactoringRequest.FIND_DEFINITION_IN_ADDITIONAL_INFO, false);
-        req.setAdditionalInfo(RefactoringRequest.FIND_REFERENCES_ONLY_IN_LOCAL_SCOPE, true);
-
-        PyReferenceSearcher pyReferenceSearcher = new PyReferenceSearcher(req);
-        pyReferenceSearcher.prepareSearch(req);
-        pyReferenceSearcher.search(req);
-        HashSet<ASTEntry> ref = pyReferenceSearcher.getLocalReferences(req);
-
-        GRAMMAR_TO_USE_FOR_PARSING = usedGrammar;
-
-        assertEquals(4, ref.size());
+        try {
+            File file = new File(TestDependent.TEST_PYSRC_TESTING_LOC + "occurrencesForDefinition/foo.py");
+            PySelection selection = new PySelection(new Document(FileUtils.getFileContents(file)), 0, 19, 4);
+            Refactorer.setPyRefactoring(new Refactorer());
+            RefactoringRequest req = new RefactoringRequest(file, selection, nature);
+            req.fillInitialNameAndOffset();
```

Comment:
Note: `req.fillInitialNameAndOffset()` is outdated with Pydev last commit (Find references not working for constant depending how it's used Fixes #PyDev-1083-`22b0f5fcdbe50898ac4ffe81751ea430f8c9ba07`) this method was renamed to `fillActivationTokenAndQualifier()`

Reply:
In this case, please rebase master into this pull request and rename this to the proper name so that it can be merged.